test/test_quantization.py

```diff
@@ -87,6 +87,7 @@
     from quantization.pt2e.test_quantize_pt2e import TestQuantizePT2E  # noqa: F401
     from quantization.pt2e.test_quantize_pt2e import TestQuantizePT2EOps  # noqa: F401
     from quantization.pt2e.test_quantize_pt2e import TestQuantizePT2EModels  # noqa: F401
+    from quantization.pt2e.test_x86inductor_quantizer import TestX86InductorQuantizePT2E  # noqa: F401
```

Comment:
maybe TestQuantizePT2EX86Inductor?

Reply:
Thanks for the suggestion. Change the name to `TestQuantizePT2EX86Inductor`.